src/index.html

```diff
@@ -22,6 +22,390 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        class="logo-link"
+        href="#"
+      >
+        <img
+          alt="Company logo"
```

Comment:
The 'alt' attribute for the logo image is set to 'Company logo', which is appropriate. However, ensure that the 'alt' text is descriptive enough for accessibility purposes.